datasets/qed_amara/qed_amara.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+The QCRI Educational Domain Corpus (formerly QCRI AMARA Corpus) is an open multilingual collection of subtitles for educational videos and lectures collaboratively transcribed and translated over the AMARA web-based platform.
+Developed by: Qatar Computing Research Institute, Arabic Language Technologies Group
+The QED Corpus is made public for RESEARCH purpose only.
+The corpus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. Copyright Qatar Computing Research Institute. All rights reserved.
+225 languages, 9,291 bitexts
+total number of files: 271,558
+total number of tokens: 371.76M
+total number of sentence fragments: 30.93M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/QED.php"
+_CITATION = """\
+A. Abdelali, F. Guzman, H. Sajjad and S. Vogel, "The AMARA Corpus: Building parallel language resources for the educational domain", The Proceedings of the 9th International Conference on Language Resources and Evaluation (LREC'14). Reykjavik, Iceland, 2014. Pp. 1856-1862. Isbn. 978-2-9517408-8-4.
```

Comment:
Can you use te bibtex format here actually ?
(I just noticed, sorry)

Feel free to create a new PR